File: tests/scripting/08_lrange.lua
-- List write + LRANGE return
-- Expected array: b, a (LPUSH inserts left to right)
redis.call('DEL', KEYS[1])
redis.call('LPUSH', KEYS[1], ARGV[1], ARGV[2])
return redis.call('LRANGE', KEYS[1], 0, -1)

File: tests/scripting_js_faithful/08_lrange.js
// Faithful copy of Lua test 08_lrange.lua
// Expected array: b, a (LPUSH inserts left to right)
redis.call('DEL', KEYS[1]);
redis.call('LPUSH', KEYS[1], ARGV[1], ARGV[2]);
return redis.call('LRANGE', KEYS[1], 0, -1);
